pkg/functions/client.go

```diff
@@ -181,7 +181,7 @@ type Instance struct {
 	Image         string            `json:"image" yaml:"image"`
 	Namespace     string            `json:"namespace" yaml:"namespace"`
 	Subscriptions []Subscription    `json:"subscriptions" yaml:"subscriptions"`
-	Labels        map[string]string `json:"labels" yaml:"labels"`
+	Labels        map[string]string `json:"labels" yaml:"labels" xml:"-"`
```

Comment:
Why only Labels require this?

Reply:
`trying to marshal unsupported type map[string]string, via x.Labels (staticcheck)`

because of the map type, im assuming by the linter message